gadgets/profile_qdisc_latency/README.mdx

```diff
@@ -0,0 +1,351 @@
+             256 -> 512        : 0        |                                        |
+             512 -> 1024       : 0        |                                        |
+            1024 -> 2048       : 0        |                                        |
+            2048 -> 4096       : 0        |                                        |
+            4096 -> 8192       : 0        |                                        |
+            8192 -> 16384      : 0        |                                        |
+           16384 -> 32768      : 0        |                                        |
+           32768 -> 65536      : 0        |                                        |
+           65536 -> 131072     : 0        |                                        |
+          131072 -> 262144     : 0        |                                        |
+          262144 -> 524288     : 0        |                                        |
+          524288 -> 1048576    : 0        |                                        |
+         1048576 -> 2097152    : 0        |                                        |
+         2097152 -> 4194304    : 0        |                                        |
+         4194304 -> 8388608    : 0        |                                        |
+         8388608 -> 16777216   : 0        |                                        |
+        16777216 -> 33554432   : 0        |                                        |
+        33554432 -> 67108864   : 0        |                                        |
+        ```
+    </TabItem>
+</Tabs>
+
+You can clean up the resources created during this guide by running the following commands:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl delete ns qdisc-latency-test
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ docker rm -f qdisc-latency-test
+        ```
+    </TabItem>
+</Tabs>
+
+## Exporting metrics
+
+The `profile_qdisc_latency` gadget can expose the histograms it generates to a
+Prometheus endpoint. To do so, you need to activate both the metrics listener as
+well as the gadget collector. To enable the metrics listener, check the
+[Exporting Metrics](../reference/export-metrics.mdx) documentation. To enable
+the collector for the `profile_qdisc_latency` gadget with the metrics name
+`qdisc-latency-metrics`, run the following command:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        WIP: Headless mode for kubectl gadget is under development
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        gadgetctl run ghcr.io/inspektor-gadget/gadget/profile_qdisc_latency:%IG_TAG% \
+                    --annotate=qdisc_latency:metrics.collect=true \
+                    --otel-metrics-name=qdisc_latency:qdisc-latency-metrics \
```

Comment:
I have not tried it, but from the discussion above it seems to need this patch:

```suggestion
                    --otel-metrics-name=qdisc:qdisc-latency-metrics \
```

Unless I misunderstood the discussion.